docs/pages/versions/unversioned/sdk/permissions.md

```diff
@@ -58,35 +93,7 @@ Determines whether your app has already been granted access to the provided perm
 
 #### Returns
 
-A `Promise` that is resolved with information about the permissions, including status, expiration, and scope (if applicable).
```

Comment:
I split this into multiple headers, under `PermissionResponse`. It's a bit hard to understand if written like this.